src/sentry/plugins/sentry_mail/models.py

```diff
@@ -195,6 +197,10 @@ def notify(self, notification):
 
         event = notification.event
 
+        Event.objects.bind_nodes([event], 'data')
```

Comment:
Do we actually need to do this here? Or was it done previously? Curious if you added this because something actually errored or if you copy/pasted from somewhere.

I ask because we already call `event.get_tags()` below, which works without any issue which indicates to me that this was working before without needing to explicitly call `bind_nodes`.

Reply:
good catch, it's not needed